Reviewer/src/main/java/com/softserveinc/reviewer/health/SimpleURLHealthCheck.java

```diff
@@ -1,37 +1,36 @@
 package com.softserveinc.reviewer.health;
 
-import com.codahale.metrics.health.HealthCheck;
 import com.google.inject.Inject;
 
 import java.net.ConnectException;
 import java.net.HttpURLConnection;
 import java.net.SocketTimeoutException;
 import java.net.URL;
+import java.util.concurrent.Callable;
 
-public class DataProvidersHealthCheck extends HealthCheck {
+public class SimpleURLHealthCheck implements Callable<Boolean> {
     private final String healthCheckUrl;
 
     @Inject
-    public DataProvidersHealthCheck(String healthCheckUrl) {
+    public SimpleURLHealthCheck(String healthCheckUrl) {
         this.healthCheckUrl = healthCheckUrl;
     }
 
     @Override
-    protected Result check() throws Exception {
+    public Boolean call() throws Exception {
         URL url = new URL(healthCheckUrl);
         HttpURLConnection connection = (HttpURLConnection) url.openConnection();
         connection.setConnectTimeout(250);
         try {
-            if(connection.getResponseCode() >= 200 && connection.getResponseCode() <= 299)
-            {
-                return Result.healthy("Service is healthy");
+            if (connection.getResponseCode() >= 200 && connection.getResponseCode() <= 299) {
```

Comment:
That should be exactly 200 response code. Other codes from range 201-209 doesn't look to be related.